netstandard/ref/System.Reflection.Emit.cs

```diff
@@ -10,13 +10,22 @@ public sealed partial class AssemblyBuilder : System.Reflection.Assembly
         public override string FullName { get { throw null; } }
         public override bool IsDynamic { get { throw null; } }
         public override System.Reflection.Module ManifestModule { get { throw null; } }
+        public void AddResourceFile(string name, string fileName) { }
```

Comment:
This should be exclude since other methods to deal with resources are excluded (like DefineResource).